lib/blacklight/solr/search_builder_behavior.rb

```diff
@@ -43,7 +43,7 @@ def add_query_to_solr(solr_parameters)
       solr_parameters[:qt] = blacklight_params[:qt] if blacklight_params[:qt]
 
       if search_field
-        solr_parameters[:qt] = search_field.qt
+        solr_parameters[:qt] = search_field.qt if search_field[:qt]
```

Comment:
Is there a reason you're using method access and hash access?

Reply:
Nope. Just surrounded by other hash-accessing.